n5/n5/Program.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using System.Linq;
+using System.Text.RegularExpressions;
+
+class Program
+{
+    static void Main(string[] args)
+    {
+        string[] text = { "Любимая! Меня вы не любили. Не знали вы, что в сонмище людском я был, как лошадь, загнанная в мыле, пришпоренная смелым ездоком." };
+        var result = FindWordsEndingInVowels(text);
+        foreach (var item in result)
+        {
+            Console.WriteLine($"{item.Key} - {item.Value}");
+        }
+
+        var client1 = new Record(1, 2022, 4, 10);
+        var client2 = new Record(1, 2022, 5, 20);
+        var client3 = new Record(2, 2021, 8, 30);
+        var client4 = new Record(2, 2021, 8, 40);
+        var client5 = new Record(3, 2022, 1, 50);
+        var client6 = new Record(3, 2021, 2, 60);
+        var database = new List<Record>() { client1,client2,client3,client4,client5,client6};
+        PrintTotalMonthsByClientAndYear(database);
+
+    }
+    static Dictionary<string, int> FindWordsEndingInVowels(string[] text) =>
```

Comment:
Я бы назвал метод немного по-другому, ведь его задача не только найти слова, но и подсчитать их частоты.
```suggestion
    static Dictionary<string, int> FindAndCountWordsEndingInVowels(string[] text) =>
```